index.html

```diff
@@ -1,11 +1,161 @@
+            row-gap: 15px;
+            border-radius: 10px;
+            background: #222;
+            color: #F6F4E4;
+            padding: 10px 20px;
+            font-style: italic;
+        }
+
+        .cardHeading {
+            font-size: 32px;
+            font-weight: 700;
+        }
+
+        .cardContent {
+            font-size: 16px;
+            font-weight: 400;
+        }
+
+        .footer {
+            width: 100%;
+            display: flex;
+            flex-direction: column;
+            gap: 20px;
+            position: absolute;
+            bottom: 0px;
+            margin: 0px;
+            padding: 40px;
+            background: #222;
+            color: #F6F4E4;
+        }
+    </style>
 </head>
+
 <body>
-    If you are not redirected, <a href="https://github.com/seriouslysean/monster-hunter-now-events">click here</a>.
+    <!-- Page Heading -->
+    <div class="heading">Monster Hunter Events Today</div>
+
+    <!-- Date -->
+    <div class="heading" id="tag_display_date"></div>
+
+    <!-- Display Event Cards Here -->
+    <div class="eventsSection" id="tag_display_events"></div>
+
+    <script type="text/javascript">
+
+        let eventsTodayList = [];
+        const TODAY = new Date();
+
+        function appendData(data) {
+            // console.log("data: ", data);
+            data.events.forEach((event) => {
+                event.dates.every((date) => {
+                    if (new Date(date.start) <= TODAY && new Date(date.end) >= TODAY) {
+                        eventsTodayList.push({
+                            name: event.summary,
+                            desc: event.description,
+                        });
+                        return false;
+                    }
+                    return true;
+                })
+            });
+
+            return eventsTodayList;
+        }
+
+        async function fetchJson() {
+            let response = await fetch('dist/events.json');
+            if (!response.ok) {
+                throw new Error("Response recieved was not ok!");
+            }
+            response = await response.json();
+            return appendData(response);
+        }
+
+
+        (async () => {
+            eventsTodayList = await fetchJson();
+        })();
+        document.getElementById("tag_display_date").innerHTML = TODAY.getDate() + "/" + TODAY.getMonth() + "/" + TODAY.getFullYear();
+
+        let tagDisplay = document.getElementById("tag_display_events");
+        // console.log(eventsTodayList.length);
+        if (eventsTodayList.length === 0) {
+            tagDisplay.innerHTML +=
+                `<div class="cardHeading">
+                            No events scheduled for today!
+                        </div>`;
+        } else {
+            eventsTodayList.forEach((event) => {
+                tagDisplay.innerHTML +=
+                    `<div class="card">
+                                <div class="cardHeading">${event.name}</div>
```

Comment:
Make sure classes are hyphened and not camel case.
```suggestion
                                <div class="card-heading">${event.name}</div>
```